org.eclipse.xtext.web/package-lock.json

```diff
@@ -1,12 +1,10 @@
 {
 	"name": "xtext",
-	"version": "2.30.0-SNAPSHOT",
 	"lockfileVersion": 2,
 	"requires": true,
 	"packages": {
 		"": {
 			"name": "xtext",
-			"version": "2.30.0-SNAPSHOT",
```

Comment:
I thought I had removed the xtext version.. maybe I had removed that from the other json file.. sorry I'm not familiar with JavaScript and the likes.